projects/aca-content/src/lib/components/search/search-results/search-results.component.html

```diff
@@ -15,20 +15,25 @@
         <div class="adf-search-results__content">
           <mat-progress-bar *ngIf="isLoading" color="primary" mode="indeterminate" aria-live="polite"> </mat-progress-bar>
           <div class="adf-search-results__content-header content">
-            <adf-search-form class="content__side--left"></adf-search-form>
-
+            <div class="content__filter-set">
+              <p>{{ 'APP.BROWSE.SEARCH.FILTER_SET' | translate }}</p>
+              <adf-search-form></adf-search-form>
+            </div>
             <mat-divider [vertical]="true" class="content__divider"></mat-divider>
-
-            <adf-search-filter-chips class="content__filter"></adf-search-filter-chips>
-
-            <button
-              mat-button
-              adf-reset-search
-              class="content__reset-action"
-              title="{{'APP.BROWSE.SEARCH.RESET_ACTION' | translate }}"
-              [attr.aria-label]="'APP.BROWSE.SEARCH.RESET_ACTION' | translate ">
-              <mat-icon> refresh </mat-icon>
-            </button>
+            <div class="content__advanced-filters">
+              <div class="content__advanced-filters--header">
```

Comment:
hmm should not we use some prefixes for classes? Like aca-?

Reply:
added prefixes

Reply:
thanks